articles/pragmatic-documentation.md

```diff
@@ -0,0 +1,97 @@
+---
+authorName: Ognjen Blagojević
+authorGithubUsername: ognjen-blagojevic-zuehlke
+issue: 86
+title: Pragmatic documentation
+---
+# {{page.title}}
+
+Writing documentation is a task that most developers and engineers just don't
+like. The reasons for this are different, but the pattern I see is that they
```

Comment:
I would use different wording here - not 'like' but maybe "... they find tiresome" or "don't see the purpose". Then you can remove the starting of the next sentence as well